tests/integration/test_cli.py

```diff
@@ -108,60 +109,53 @@ def test_pipenv_graph(pipenv_instance_pypi):
 @pytest.mark.cli
 def test_pipenv_graph_reverse(pipenv_instance_private_pypi):
```

Comment:
rewrote the test to use the json output since its going to be way less flaky going forward doing it that way.

Reply:
Now that pipdeptree supports it, it does makes sense!